### ISSUE TITLE
Text too big for mobile devices and footer not responsive

### ISSUE THREAD

**Issue by kedar1998** [amupedia2021/Project-Amupedia#192]
I will convert text to mobile friendly text and fix footer. Please assign me this issue under GSSOC'23

![image](https://github.com/amupedia2021/Project-Amupedia/assets/103363619/a0de1ddd-92f4-4ef0-984f-bd45678e97bc)

![image](https://github.com/amupedia2021/Project-Amupedia/assets/103363619/00125f73-a654-42f4-a0c3-08d61ad294f8)

![image](https://github.com/amupedia2021/Project-Amupedia/assets/103363619/4865fc97-2716-4557-8be7-0d5b578719b9)

**Comment by anmode** [comment]
Ok could you do mobile optimization then?? footer got fixed

**Comment by kedar1998** [comment]
Yes!! I would love to work on this. Can you please assign it to me under GSSOC'23.
